test/e2e/retry.go

```diff
@@ -51,5 +45,5 @@ func retryWithExponentialBackOff(fn func() error) error {
 			return false, err
 		}
 	}
-	return wait.ExponentialBackoff(backoff, retryFn(fn))
+	return wait.PollImmediate(interval, timeout, retryFn(fn))
```

Comment:
is this timeout being applied to the overall time retrying or to each attempt?

Reply:
The `interval` parameter enforces the wait time between retries, the `timeout` parameter enforces the overall timeout (inclusive of retries).